frontend/app/events/eventDetail/eventDetail.component.spec.ts

```diff
@@ -43,11 +51,16 @@ describe("EventDetailComponent", () => {
         });
     });
 
-    it("Should set a new active Event", inject([EventDetailComponent], (evDComponent) =>  {
+    it("Should set a new active & not covered Event", inject([EventDetailComponent], (evDComponent) =>  {
```

Comment:
+ Test, dass eine Tweets angezeigt werden (siehe `Should set a new active & covered Event`)